pp_ctl.c

```diff
@@ -3906,7 +3906,10 @@ S_require_file(pTHX_ SV *sv)
 	/* reuse the previous hv_fetch result if possible */
 	SV * const * const svp = svp_cached ? svp_cached : hv_fetch(GvHVn(PL_incgv), unixname, unixlen, 0);
 	if ( svp ) {
-	    if (*svp != &PL_sv_undef)
+            /* we already did a get magic if this was cached */
+            if (!svp_cached)
+                SvGETMAGIC(*svp);
```

Comment:
These 3 lines appear to suffer an extra level of indentation.

Reply:
Or it might just be github interface doing something suboptimal when showing tabs v. spaces, never mind me.